compiler/sem/semtypes.nim

```diff
@@ -1455,6 +1459,17 @@ proc semProcTypeNode(c: PContext, n, genericParams: PNode,
             a[^2].info,
             reportTyp(rsemMisplacedMagicType, typ))
 
+      template isUntyped(t: PType): bool =
+        t.kind == tyUntyped or
+          t.kind == tyVarargs and t.len > 0 and t[0].kind == tyUntyped
+
+      if untypedParamPos == 0:
+        if typ.isUntyped:
```

Comment:
```suggestion
        if isUntyped(typ):
```
This makes skimming this part a little bit easier, as both `isUntyped` calls use the the same syntax.

Reply:
I switched it for the negation case because that read a fair bit better, but thought this might be ok. I'll change it, thanks.